Tools/autotest/param_metadata/rstemit.py

```diff
@@ -263,7 +263,7 @@ def emit(self, g):
                         ret += self.render_table_headings(ret, row, headings, field_table_info, field, param)
                     elif field == "Range":
                         (param_min, param_max) = (param.__dict__[field]).split(' ')
-                        row.append("%s - %s" % (param_min, param_max,))
+                        row.append("%s / %s" % (param_min, param_max,))
```

Comment:
Actually I would prefer this for the wiki...however, any change would probably break any GCS's parsing of the range for display, although all except QGC do a horrible job of displaying metadata....MP displays nothing for many many params....same with MAVProxy's param editor....I think metadata display in GCS deserves a big meeting/discussion IMHO
```suggestion
                        row.append("%s to %s" % (param_min, param_max,))
```

Reply:
I doubt a GCS is parsing the restructured text for param values. This would only be fore the Parameters.rst and ParametersLatex.rst and both would only receive a visual change.

Reply:
in taht case, I think english "to" is best presentation

Reply:
Yep, I think will be used only by the wiki or printed, so it should be fine